Mod2/Lessons/Week4/CRUDwithORM.md

```diff
@@ -0,0 +1,182 @@
+# CRUD with ORM
+
+## Intro
+
+Up to this point, you�ve created a database using PostGreSQL and pgAdmin. You�ve used CRUD operations to manipulate your database:
+
+-   (C)reate new data using the INSERT statement
+-   (R)ead existing data using the SELECT statement
```

Comment:
We could also call out that statements like `WHERE` `ORDERBY` and `JOIN` also help Read the data.

Reply:
Done!